hmt_escrow/job.py

```diff
@@ -315,14 +321,14 @@ def launch(self, pub_key: bytes) -> bool:
 
         job_addr = self._last_escrow_addr()
         LOG.info("Job's escrow contract deployed to:{}".format(job_addr))
-        self.job_contract = get_escrow(job_addr)
+        self.job_contract = get_escrow(job_addr, self.hmt_server_addr)
 
         (hash_, manifest_url) = upload(self.serialized_manifest, pub_key)
         self.manifest_url = manifest_url
         self.manifest_hash = hash_
         return self.status() == Status.Launched and self.balance() == 0
 
-    def setup(self, gas: int = GAS_LIMIT, sender: str = None) -> bool:
```

Comment:
This is not a backward-compatible API change. Due to this, a major part of the package version should be changed